src/scs/slsa-framework.md

```diff
@@ -0,0 +1,22 @@
+# SLSA Framework
+
+Supply chain Levels for Software Artifacts (SLSA) is a security framework for tampering prevention, integrity improvement and securing packages and infrastructure of a project.
```

Comment:
... tampering prevention, integrity improvement, and securing packages and infrastructure of a project.

Should we add this link https://slsa.dev/ somewhere?

Reply:
Yes, the link should (must) be added, but where to?

Reply:
Ok, let's think about it when others sections will be ready.